float8_experimental/float8_tensor.py

```diff
@@ -115,7 +133,13 @@ def to_fp8_no_autograd(
             stride=bits_fp8.stride(),
         )
 
-    return Float8Tensor(bits_fp8, x_scale, x.dtype, mm_config=mm_config)
+    return Float8Tensor(
+        bits_fp8,
+        x_scale,
```

Comment:
just curious, since we decided to not add `scaling_strategy` to `torch._scaled_mm`, why do we need it here?

Reply:
We could make this a property of Float8Tensor, e.g. infer from the existing scales... hmm 
Actually I might like this more.. 
We need the enum still since we want modules to specify their granularity